### ISSUE TITLE
[amplify_analytics_pinpoint] Update `device_info_plus` dependency

### ISSUE THREAD

**Issue by asaarnak** [aws-amplify/amplify-flutter#5565]
Increase `device_info_plus` from `^10.0.1` to `^11.0.0`
```
Because amplify_auth_cognito >=2.4.1 depends on amplify_analytics_pinpoint >=2.4.1 <2.5.0 which depends on device_info_plus ^10.0.1, amplify_auth_cognito >=2.4.1 requires device_info_plus
  ^10.0.1.
So, because my_app depends on both amplify_auth_cognito ^2.4.1 and device_info_plus ^11.0.0, version solving failed.
```

**Comment by asaarnak** [comment]
@NikaHsn Any updates on that issue?

**Comment by asaarnak** [comment]
Here is a duplicate issue: https://github.com/aws-amplify/amplify-flutter/issues/5721

**Comment by ekjotmultani** [comment]
Hi @asaarnak , we are looking to get this out in the next release. Thank you for the bump on this issue

**Comment by asaarnak** [comment]
```
Because amplify_auth_cognito_dart 0.11.8 depends on win32_registry ^1.1.0 and no versions of amplify_auth_cognito_dart match >0.11.8 <0.12.0, amplify_auth_cognito_dart ^0.11.8 requires win32_registry ^1.1.0.
And because device_info_plus >=11.3.1 depends on win32_registry ^2.0.1, device_info_plus >=11.3.1 is incompatible with amplify_auth_cognito_dart ^0.11.8.
And because amplify_auth_cognito 2.6.0 depends on amplify_auth_cognito_dart ^0.11.8 and no versions of amplify_auth_cognito match >2.6.0 <3.0.0, device_info_plus >=11.3.1 is incompatible with amplify_auth_cognito ^2.6.0.
So, because my_app depends on both amplify_auth_cognito ^2.6.0 and device_info_plus ^11.3.1, version solving failed.
```
New error, because:
`device_info_plus: ^11.3.1` updated major `win32_registry` version from from `^1.1.5` to `^2.0.1`

**Comment by ekjotmultani** [comment]
Hi @asaarnak, thanks, we will look at win32_registry, seems like the jump from 1 to 2 has lots of breaking changes so that could take some time and investigation. In the spirit of transparency, the reason the device_info_plus dependency has taken a while to get to is because we have had a whole nest to untangle [upgrading drift](https://github.com/aws-amplify/amplify-flutter/pull/5741/files#), however now that that PR is out we should be able to get a move on upgrading device_info_plus.

**Comment by asaarnak** [comment]
Hei @tyllark 
Any news about this issue?

**Comment by ekjotmultani** [comment]
Hi @asaarnak, sorry we missed replying back on this issue, but this dependency was updated as of Amplify Flutter 2.6.1. Please try updating!

**Comment by nikorehnback** [comment]
> Hi [@asaarnak](https://github.com/asaarnak), sorry we missed replying back on this issue, but this dependency was updated as of Amplify Flutter 2.6.1. Please try updating!

For me it shows in 2.6.1 pubspec:

device_info_plus: ^10.0.1

@ekjotmultani am I missing something here?

**Comment by asaarnak** [comment]
Seems like this is not released yet.
We are getting this error:
```
Because amplify_auth_cognito_dart 0.11.9 depends on win32_registry ^1.1.0 and no versions of amplify_auth_cognito_dart match >0.11.9 <0.12.0, amplify_auth_cognito_dart ^0.11.9 requires win32_registry ^1.1.0.
And because device_info_plus >=11.3.2 depends on win32_registry ^2.1.0, device_info_plus >=11.3.2 is incompatible with amplify_auth_cognito_dart ^0.11.9.
And because amplify_auth_cognito 2.6.1 depends on amplify_auth_cognito_dart ^0.11.9 and no versions of amplify_auth_cognito match >2.6.1 <3.0.0, device_info_plus >=11.3.2 is incompatible with amplify_auth_cognito ^2.6.1.
So, because my_app depends on both amplify_auth_cognito ^2.6.1 and device_info_plus ^11.4.0, version solving failed.
```

**Comment by tyllark** [comment]
Hello @asaarnak  and @nikorehnback. The changes have been merged into our main branch but have not been released yet. I'm sorry for the delay, I will start putting together a release and will post back here once it is live.

**Comment by asaarnak** [comment]
Hei @tyllark. 
This seems to be fixed now.
But a new patch version is giving us `built_value 8.10.1` error now.

```
Because no versions of amplify_auth_cognito match >2.6.2 <3.0.0 and amplify_auth_cognito 2.6.2 depends on amplify_auth_cognito_dart ^0.11.10, amplify_auth_cognito ^2.6.2 requires amplify_auth_cognito_dart ^0.11.10.
Because amplify_auth_cognito_dart 0.11.10 depends on built_value 8.10.0 and no versions of amplify_auth_cognito_dart match >0.11.10 <0.12.0, amplify_auth_cognito_dart ^0.11.10 requires built_value 8.10.0.
Thus, amplify_auth_cognito ^2.6.2 requires built_value 8.10.0.
So, because my_app depends on both built_value ^8.10.1 and amplify_auth_cognito ^2.6.2, version solving failed.
```

**Comment by tyllark** [comment]
Hello @asaarnak, I'm going to close this issue as the device_info_plus is resolved. As a heads up the [built_value](https://github.com/aws-amplify/amplify-flutter/issues/6156) fix was just released too!
